src/Exporter/DataItemMatchFinder.php

```diff
@@ -57,6 +57,12 @@ public function matchExpElement( ExpElement $expElement ) {
 			return $dataItem;
 		}
 
+		$dataItem = $expElement->getDataItem();
+		if ($dataItem && $dataItem instanceof DIWikiPage) {
```

Comment:
Need spaces to match code style and an explicit check against null or false or whatever the value might be:

`if ( $dataItem !== null && $dataItem instanceof DIWikiPage ) {`

Reply:
thanks, updated accordingly